### ISSUE TITLE
Sort posts by published rather than by creation of draft

### ISSUE THREAD

**Issue by jefflam** [natew/obtvse#73]
Hi NateW,

First off, I just want to say thank you for creating this 'replica' of svbtle. I would have created it for myself months back if you had not created it, as I too love the way that it is so distract-free. So, thank you on that!

Next, my coding skills isn't that great, still learning. However, throughout my weeks of using obtvse, I noticed that posts are sorted via the creation date of the post (draft or otherwise). I understand you simple switch 'on' or 'off' the draft property for the posts model, which can make it tricky for me to implement sorting of Posts by draft instead of creation date. 

I ask this because I usually type in the draft random ideas or what not, and generally only publish them after a certain period of time depending on my mood of writing. Meanwhile, I have written and 'published' other posts. Thus, when I publish the post from before, it appears 'hidden'. 

Any ideas?

Thanks!

**Comment by natew** [comment]
Yes its a known bug, for now I just copy it into a new post haha.  I'm working on a solution on the refresh branch but its not done yet.

Nate
natewienert.com

**Comment by jefflam** [comment]
Ah, that's a good hacking solution. :) 

Alright, I'll await your solution then ;)

Thanks!
